22-discovery.ks

```diff
@@ -56,6 +56,7 @@ sed -i '/\[Unit\]/a ConditionPathExists=/etc/NetworkManager/system-connections/p
 sed -i '/\[Service\]/a EnvironmentFile=-/etc/default/discovery' /usr/lib/systemd/system/foreman-proxy.service
 sed -i '/\[Service\]/a ExecStartPre=/usr/bin/generate-proxy-cert' /usr/lib/systemd/system/foreman-proxy.service
 sed -i '/\[Service\]/a PermissionsStartOnly=true' /usr/lib/systemd/system/foreman-proxy.service
+sed -i '/\[Service\]/a TimeoutStartSec=9999' /usr/lib/systemd/system/foreman-proxy.service
```

Comment:
This is to prevent killing foreman-proxy when it is waiting for IPv4. Although systemd supports "infinity" here, version in RHEL 7.4 does not have it, thus 9999.